charts/apisix/templates/deployment.yaml

```diff
@@ -45,6 +45,7 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      serviceAccountName: {{ include "apisix.serviceAccountName" . }}
```

Comment:
Check this field before using it.

Reply:
I belief the helper template already included the necessary checking? If serviceAccount is not created, it will use the default "default" service account. I see most of the helm charts using it directly.

Reply:
Oh, got it, thanks!